lib/Config.uk

```diff
@@ -32,3 +32,13 @@ config HAVE_SYSCALL
 config HAVE_X86PKU
 	bool
 	default n
+
+##
+# Compatibility entries for legacy libraries
+##
+
+config LIBPOSIX_EVENT
+	bool
+	select LIBPOSIX_POLL
+	select LIBPOSIX_EVENTFD
+	default n
```

Comment:
In general `default n` we never need to type because that is the default.

Reply:
Removed. A future PR should delete all those `default n` lines above (and in the rest of the Unikraft codebase, I don't think I've ever seen a config without a default n)